### PR TITLE
Update agent_integration.md

### PR DESCRIPTION
**Pull request by emarsha94**
to add how to install wheels on container env

### What does this PR do? What is the motivation?
to add how to install wheels on container env

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/developers/integrations/agent_integration.md [hunk 1]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in containerized environments: 
+1. Mount the whl into a initContainer
+2. Run the whl install in a init container
```
Comment (emarsha94) [inline content/en/developers/integrations/agent_integration.md:441]:
```suggestion
2. Run the whl install in the init container
```

File: content/en/developers/integrations/agent_integration.md [hunk 2]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in containerized environments: 
+1. Mount the whl into a initContainer
+2. Run the whl install in the init container
+3. Mount the init container in the agent container
```
Comment (emarsha94) [inline content/en/developers/integrations/agent_integration.md:442]:
```suggestion
3. Mount the init container in the agent container while it's running
```

File: content/en/developers/integrations/agent_integration.md [hunk 3]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in containerized environments: 
+1. Mount the whl into a initContainer
+2. Run the whl install in the init container
+3. Mount the init container in the agent container while it's running
```
Comment (hestonhoffman) [inline content/en/developers/integrations/agent_integration.md:442]: We're gonna need some commands for these.

Comment (emarsha94) [reply content/en/developers/integrations/agent_integration.md:442]: I think this is something they configure, not something we instruct them to already have. Agent would be best to loop in for commands if we need them @hestonhoffman

File: content/en/developers/integrations/agent_integration.md [hunk 4]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in containerized environments:
```
Comment (emarsha94) [inline content/en/developers/integrations/agent_integration.md:439]:
```suggestion
For installing your wheel to test in Kubernetes environments: 
```

File: content/en/developers/integrations/agent_integration.md [hunk 5]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
   ```
 </details>
 
+For installing your wheel to test in Kubernetes environments: 
+1. Mount the whl into a initContainer
```
Comment (hestonhoffman) [inline content/en/developers/integrations/agent_integration.md:440]:
```suggestion
1. Mount the `.whl` file into an initContainer.
```

File: content/en/developers/integrations/agent_integration.md [hunk 6]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
...
+2. Run the whl install in the init container
```
Comment (hestonhoffman) [inline content/en/developers/integrations/agent_integration.md:441]:
```suggestion
2. Run the wheel install in the initContainer.
```

File: content/en/developers/integrations/agent_integration.md [hunk 7]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
...
+3. Mount the init container in the agent container while it's running
```
Comment (hestonhoffman) [inline content/en/developers/integrations/agent_integration.md:442]:
```suggestion
3. Mount the initContainer in the Agent container while it's running.
```

File: content/en/developers/integrations/agent_integration.md [hunk 8]
```diff
@@ -436,6 +436,12 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
...
+
+For customer install commands for both host and container environments, please see [here.][35]
```
Comment (hestonhoffman) [inline content/en/developers/integrations/agent_integration.md:444]:
```suggestion
For customer install commands for both host and container environments, see the [Community and Marketplace Integrations documentation][35].
```

**Review by hestonhoffman — COMMENTED** [review]
Hey @emarsha94, sorry this took so long to review. I left some small edits just for consistency. I'd like to circle back later with a follow-up PR to give some example commands.

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**